_posts/2022-09-04-configuring-security-context-in-pods-managed-by-strimzi.md

```diff
@@ -0,0 +1,185 @@
+You can configure the Pod Security Provider in the Cluster operator configuration using the environment variable `STRIMZI_POD_SECURITY_PROVIDER_CLASS`.
+Strimzi provides two different providers:
+* Baseline provider
+* Restricted provider
+
+These providers correspond to the _baseline_ and _restricted_ profiles as defined in the [Kubernetes Pod Security Standards](https://kubernetes.io/docs/concepts/security/pod-security-standards/).
+
+### Baseline provider
+
+The Baseline provider is the default provider which is used when you don't configure any other provider.
+If you want, you can also enable it explicitly by setting the `STRIMZI_POD_SECURITY_PROVIDER_CLASS` environment variable to `io.strimzi.plugin.security.profiles.impl.BaselinePodSecurityProvider`.
+Alternatively, you can also use the _shortcut_ `baseline`:
+
+```yaml
+# ...
+env:
+  # ...
+  - name: STRIMZI_POD_SECURITY_PROVIDER_CLASS
+    value: baseline
+  # ...
+```
+
+The Baseline provider configures the Strimzi managed pods according to the [Kubernetes _baseline_ profile](https://kubernetes.io/docs/concepts/security/pod-security-standards/#baseline).
+It also provides backwards compatibility because it configures the pods in exactly the same way as previous Strimzi versions.
+So with this provider, there will be no change to how the Strimzi-managed pods are configured.
+
+### Restricted provider
+
+The Restricted provider can be enabled by setting the `STRIMZI_POD_SECURITY_PROVIDER_CLASS` environment variable to `io.strimzi.plugin.security.profiles.impl.RestrictedPodSecurityProvider`.
+Or you can also use the _shortcut_ keyword `restricted`:
+
+```yaml
+# ...
+env:
+  # ...
+  - name: STRIMZI_POD_SECURITY_PROVIDER_CLASS
+    value: restricted
+  # ...
+```
+
+The Restricted provider configures the Strimzi managed pods according to the [Kubernetes _restricted_ profile](https://kubernetes.io/docs/concepts/security/pod-security-standards/#restricted).
+The _restricted_ profile reduces the number of different permissions your applications can use:
+* Restricts the volume types which can be used
+* Does not allow privilege escalation
+* Requires applications to run under a non-root user
+* The _Seccomp_ profile must be set to `RuntimeDefault` or `Localhost`
+* Containers must drop all capabilities and can use only the `NET_BIND_SERVICE` capability
+
+All of these requirements are implemented in our restricted provider.
+When you enable it in the Cluster Operator, all the containers it creates will have the following security context set:
+
+```yaml
+# ...
+securityContext:
+  allowPrivilegeEscalation: false
+  capabilities:
+    drop:
+      - ALL
+  runAsNonRoot: true
+  seccompProfile:
+    type: RuntimeDefault
+# ...
+```
+
+Any Pods and containers which will be already deployed and running when you enable the restricted provider will be rolled and the security context will be set for them as well.
```

Comment:
```suggestion
Any Pods and containers that are already deployed and running when you enable the restricted provider are rolled so that the security context is set for them as well.
```